aws/lambda/pytorch-auto-revert/pytorch_auto_revert/__main__.py

```diff
@@ -100,6 +100,11 @@ def get_opts() -> argparse.Namespace:
         action="store_true",
         help="When restarts complete and secondary pattern matches, log REVERT",
     )
+    workflow_parser.add_argument(
```

Comment:
there is already a global parameter `--dry-run`. 

There is no need to add this guy here :)